datasets/librispeech_preprocess.py

```diff
@@ -167,9 +167,11 @@ def run(input_dir, output_dir, tokenizer_vocab_path):
   ]
   for subset in subset_list:
     logging.info('Processing split = %s...', subset)
-    subset_dir = os.path.join(output_dir, subset)
-    os.makedirs(subset_dir, exist_ok=True)
-    example_ids, num_entries = preprocess_data(subset_dir, tokenizer, subset)
+    in_dir = os.path.join(input_dir, subset)
+    out_dir = os.path.join(output_dir, subset)
+    os.makedirs(out_dir, exist_ok=True)
+    example_ids, num_entries = preprocess_data(in_dir, output_dir, tokenizer,
+    subset)
```

Comment:
nit: fix the arg formatting, something like:
```
example_ids, num_entries = preprocess_data(
    in_dir, output_dir, tokenizer, subset)
```

Reply:
Done!